source/products/hansemerkur/softfair/rest-api.rst

```diff
@@ -0,0 +1,524 @@
+            },
+            "quote_token": "<QUOTE_TOKEN>"
+        }'
+
+Example Response
+~~~~~~~~~~~~~~~~
+
+.. code:: javascript
+
+    {
+        "id": "<LEAD_ID>",
+        "livemode": false,
+        "firstname": "firstName",
+        "lastname": "lastName",
+        "email": "test@kasko.io",
+        "quote": {
+            "version": 2,
+            "customer_input": {
+                "gross_premium": "100000",
+                "policy_start_date": "2020-01-01",
+                "policy_end_date": "2020-02-01"
+            },
+            "touchpoint_id": "<TOUCHPOINT_ID>",
+            "item_id": "<ITEM_ID>",
+            "subscription_plan_id": "<SUBSCRIPTION_ID>",
+            "gross_payment_amount": 100000,
+            "payment_data": {
+                "gross_premium": 100000,
+                "premium_tax": 15966,
+                "net_premium": 84034,
+                "net_net_premium": 62185,
+                "net_commission_total": 21849,
+                "tax_rate": 0.19
+            },
+            "data": [],
+            "duration_strategy": "fixed_start_and_end_date",
+            "duration_data": {
+                "policy_start_date": "2019-12-31T23:00:00+00:00",
+                "policy_end_date": "2020-01-31T23:00:00+00:00"
+            },
+            "billing_cycles": 1,
+            "quote_created_at": "2020-09-17T09:29:02+00:00",
+            "signature": "<SIGNATURE>"
+        },
+        "data": {
+            "dob": "1993-12-02",
+            "policy_start_date": "2020-01-01",
+            "policy_end_date": "2020-02-01",
+            "dog_name": "dogName",
+            "dog_hybrid": "no",
+            "dog_breed": "doberman",
+            "dog_dangerous": "no",
+            "registration": "yes",
+            "coverage": "no",
+            "deductible": "yes",
+            "payment_method": "credit card",
+            "payment_frequency": "monthly",
+            "annual_gross_premium": "10000000",
+            "access_pin": "VPE3TEFH"
+        },
+        "product_id": "<ITEM_ID>",
+        "policy_id": null,
+        "touchpoint_id": "<TOUCHPOINT_ID>",
+        "distributor_id": "<DISTRIBUTOR_ID>",
+        "referrer_url": null,
+        "whitelisted_referrer_url": null,
+        "metadata": null,
+        "opt_out_date": null,
+        "language": "de",
+        "currency": "EUR",
+        "updated_at": "2020-09-17T12:40:09+00:00",
+        "created_at": "2020-09-17T12:40:09+00:00",
+        "assets": [],
+        "token": "<TOKEN>>",
+        "status": "pending",
+        "_links": {
+            "_self": {
+                "href": "https://api.kasko.io/leads/<LEAD_ID>"
+            },
+            "distributor": {
+                "href": "https://api.kasko.io/accounts/<DISTRIBUTOR_ID>"
+            },
+            "product": {
+                "href": "https://api.kasko.io/products/<ITEM_ID>"
+            }
+        }
+    }
+
+Show Lead
+~~~~~~~~~
+Show lead data
```

Comment:
Are these three lines necessary?
```
Show Lead
~~~~~~~~~
Show lead data
```

Reply:
Removed the 3rd line, first two I think are necessary to see where a new request begins